org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/dropwizard/xtext/XtextApplication.xtend

```diff
@@ -40,8 +44,9 @@ abstract class XtextApplication<T extends XtextConfiguration> extends Dropwizard
 		modules += [ binder |
 			binder.bind(AbstractFileSystemAccess).to(JavaIoFileSystemAccess).asEagerSingleton
 			binder.bind(IJavaCompiler).to(EclipseJavaCompiler)
-			binder.bind(IIssueHandler).to(DefaultIssueHandler)
+			binder.bind(IIssueHandler).to(ValidationMarkerUpdater)
 			binder.bind(IndexUpdater).to(GradleProjectIndexUpdater)
+			binder.bind(ResponseBuilder).toProvider[Response.ok]
```

Comment:
Why do I need this provider here?

Reply:
`ValidationMarkerResource` relies on it. This was to make the class more testable – a simple wrapper around that static method, so that it can be mocked out the usual way. Unfortunately, there is no method that returns a `ResponseBuilder` without any fields set initially. With this particular binding, HTTP status code "OK" will be pre-set by default, which I think is reasonable.